presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/TableScanMatcher.java

```diff
@@ -64,7 +80,19 @@ public MatchResult detailMatches(PlanNode node, PlanNodeCost cost, Session sessi
         String actualTableName = tableMetadata.getTable().getTableName();
         return new MatchResult(
                 expectedTableName.equalsIgnoreCase(actualTableName) &&
-                        domainMatches(tableScanNode, session, metadata));
+                        domainMatches(tableScanNode, session, metadata) &&
+                        originalConstraintMatches(tableScanNode));
+    }
+
+    private boolean originalConstraintMatches(TableScanNode node)
+    {
+        Map<String, SymbolReference> assignments = node.getOutputSymbols().stream()
+                .collect(toMap(Symbol::getName, Symbol::toSymbolReference));
```

Comment:
we agreed that we'll do it this way, as it's way easier for the end-user than providing the needed `.withAlias` invocations. In fact, we weren't even able to make this work when reusing the upstream `SymbolAliases`, as they only flow up the matching process and aren't available for the latter `Matcher`s of same `PlanMatchPattern`